### ISSUE TITLE
why limiting to localhost only?

### ISSUE THREAD

**Issue by garcimouche** [chbatey/kafka-unit#36]
Thanks for this useful project. I need to run an integration test from a kafka consumer which is deployed to a running docker container with [arquillian](http://arquillian.org/). I have issues accessing the kafka-unit server which lives itself outside of the docker container (in localhost). The only way to get this work is to start the kafka server on the docker bridge interface as well as the consumer (in manual testing).

**Comment by chbatey** [comment]
Makes sense. Happilly accept a PR for making the listen address configurable
